### DISCUSSION TITLE
Commerce 20240907 Zhong-Yuan cultural creative park stage performance enrollment plan

### DISCUSSION THREAD

**Discussion by Sean880128** [J-tour/J-tour.github.io#27] [General]
![image](https://github.com/J-tour/J-tour.github.io/assets/163014964/92b32296-c22a-4cdc-ac29-6f7f2a7d094f)
9/7(六) , 9/8(日) , 9/14(六) , 9/15(日)
演出時間50~60分鐘含架場,無PA大叔，需要自己調
演出費用20,000元
報名截止日期 : 4/15 23:59PM

**Comment by ShengHaoLo** [comment]
[Sean](@Sean880128 )有沒有這個案件的報名截止日與連結可以參考呢？
